docs/articles/documentation/using-testcafe/installing-testcafe.md

```diff
@@ -5,50 +5,67 @@ permalink: /documentation/using-testcafe/installing-testcafe.html
 ---
 # Installing TestCafe
 
-You can install TestCafe through `npm` globally or locally in your project.
+You can install TestCafe from `npm` globally or locally in your project.
 
-* [Globally](#globally)
-* [Locally](#locally)
+[Global installation](#global-installation) is required to run TestCafe in the [command line](command-line-interface.md) from any location. For instance, you need to install TestCafe globally to execute the `testcafe` command from your custom application.
 
-## Globally
+> You can use `npx`, `yarn run` or the `npm` scripts to run the `testcafe` command from the project directory without global installation.
+
+[Local installation](#local-installation) should be preferred for continuous integration systems, [Node.js applications](programming-interface/README.md) and other scenarios where global installation is not required.
+
+* Local installation makes your project setup easier: `npm install` executed in the project directory installs all dependencies including TestCafe.
+* Different projects can use different local TestCafe versions.
+
+## Global Installation
 
 ```bash
 npm install -g testcafe
 ```
 
-After that, you can run TestCafe from a [command line](command-line-interface.md) by using the `testcafe` command.
+After that, use the [testcafe](command-line-interface.md) command to run TestCafe from the command line.
 
 ```bash
 testcafe chrome tests/
 ```
 
-## Locally
+## Local Installation
 
-The following command will install TestCafe into your project directory and save it on the dependencies list.
+The following command installs TestCafe into your project directory and saves it on the dependencies list.
 
 ```bash
 npm install --save-dev testcafe
 ```
 
-The local installation is preferred for continuous integration scenarios or if you are going to use TestCafe from a Node.js application (see [Programming Interface](programming-interface/README.md)). This kind of installation has two advantages.
+To run a local TestCafe version from the [command line](command-line-interface.md), use one of the following methods:
 
-* It makes your project setup easier: running `npm install` in the project directory will automatically install TestCafe as well.
-* Different projects can depend on different versions of TestCafe.
+* the [npx](https://www.npmjs.com/package/npx) command:
```

Comment:
Actually, you can use `npx testcafe` without explicitly installing TestCafe locally. It would be nice if we can mention it.

Reply:
However, it is definitely not a recommended way to use TestCafe, since if you don't have any TestCafe installation, each time you run `npx testcafe` it will reinstall TestCafe and all its dependencies.